test/paper-dialog-scrollable.html

```diff
@@ -219,6 +235,36 @@
           });
         });
 
+        test('shows bottom divider if content is added dynamically', function(done) {
+          var container = fixture('dynamic-content');
```

Comment:
This could be much simpler, e.g. use the `basic` fixture and remove the content:
```js
test('removes bottom divider if content is removed dynamically', function(done) {
  var container = fixture('basic');
  var scrollable = Polymer.dom(container).querySelector('paper-dialog-scrollable');
  assert.isTrue(scrollable.classList.contains('scrolled-to-bottom'), 'scrollable incorrectly has scrolled-to-bottom class');    
  setTimeout(function() {
    Polymer.dom(container).innerHTML = '<p>hi</p>';
    scrollable.updateScrollState();
    assert.isFalse(scrollable.classList.contains('scrolled-to-bottom'), 'scrollable correctly has scrolled-to-bottom class');
    assert.ok(getComputedStyle(scrollable, '::after').content, '::after has content');
    done();
  }, 10);
});
```
WDYT?